src/core/http/request_handler.rs

```diff
@@ -321,7 +321,13 @@ async fn handle_request_inner<T: DeserializeOwned + GraphQLRequestLike>(
 
             graphql_request::<T>(req, &Arc::new(app_ctx), req_counter).await
         }
-
+        hyper::Method::GET if req.uri().path() == "/status" => {
+            let status_response = Response::builder()
+                .status(StatusCode::OK)
+                .header(CONTENT_TYPE, "application/json")
+                .body(Body::from(r#"{"message": "ready"}"#))?;
```

Comment:
Could you please include a unit test in the same file?

Reply:
yeah, sure.